### PR TITLE
Change markdown files to create PDF using pandoc

### PR DESCRIPTION
**Pull request by hosseinn**
Pandoc can now be used to create PDF out of the Markdown files

    pandoc --toc `cat toc.txt` -o inside-libreoffice.pdf

The toc.txt file should contain list of files in the correct order. A
rename that removes the space character in the file names can help to
simplify the pandoc invoke. This rename will be done later.

### CODE REVIEW

**Comment by flywire** [comment]
PR supported subject to the following:
1. After V1 release
2. Must be compatible with https://github.com/flywire/lo-p/issues/1#issuecomment-950305502
    > 5: [Tab](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/) code examples by language
3. Can a single source be used for Contributors/Authors?
    - update with your attribution
4. > removes the space character in the file names
    - only occurs in part divider names, rename now or find better fix

I can't test for a few days.

File: docs/27-Funcs_Analysis.md [hunk 1]
```diff
@@ -1421,11 +1421,12 @@ to Calc's solvers.
...
-```java
-x <= 6
-y <= 8
-z >= 4
-```
+=== "java"
+    ```java
+    x <= 6
+    y <= 8
+    z <= 4
```
Comment (hosseinn) [inline docs/27-Funcs_Analysis.md:1428]: Is this change (z >=4 -> z <=4 ) intentional?

Comment (flywire) [reply docs/27-Funcs_Analysis.md]: No, manual conflict resolution instead of scripted change.

**Comment by flywire** [comment]
> Pandoc can now be used to create PDF out of the Markdown files

Can you update https://github.com/flywire/lo-p/wiki for Pandoc? Just add enough to install and generate output.

**Pull request merged**